<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game of Life</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="gridLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <layout class="QHBoxLayout" name="buttonsLayout">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stepButton">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="saveLoadLayout">
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="patternButton">
            <property name="text">
             <string>Load Pattern</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="slidersLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sizeSpinBox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sizeSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Interval (ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="intervalSpinBox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string extracomment=" "/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="intervalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
    <addaction name="actionStep"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuControl"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset>
     <normaloff>png/play128.png</normaloff>png/play128.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
